packages/calcite-components/src/components/tabs/tabs.e2e.ts

```diff
@@ -258,11 +240,25 @@ describe("calcite-tabs", () => {
         document.body.innerHTML = `<${wrapperName}></${wrapperName}>`;
 
         const wrapper = document.querySelector(wrapperName);
+
+        async function waitForAnimationFrames(count) {
```

Comment:
We already have a [helper to wait an animation frame](https://github.com/Esri/calcite-design-system/blob/main/packages/calcite-components/src/tests/utils.ts#L255-L257). Can you use that here?

Reply:
Opening a follow-up refactor Pr for this one.